tern/formats/spdx/spdxtagvalue/generator.py

```diff
@@ -143,6 +143,9 @@ def format_license(license_string):
     We will split up the licenses by spaces, prepend each string with a
     "LicenseRef-" and then join the strings with an " AND "'''
     license_list = get_package_licenses(license_string)
+    for l in license_list:
+        if 'and' in l or 'or' in l:
+            license_list.remove(l)
```

Comment:
How about using `re.sub` in case there are more unacceptable words we will have to deal with in the future?

Reply:
@nishakm I'm confused how I would use `re.sub` here. Do you mean use `re.search` to find the elements in the list? Not sure how re.sub would help me here as I want to compare the full license string and if it matches completely with and or or, remove the complete license element from the list. I think re.sub will manipulate just the string. But if we manipulate just the license string, even by substituting it with an empty string it would end up as an empty element in the list which we wouldn't want either. Thanks in advance :)

Reply:
Oh I see what you mean. So I guess what's going on is that `get_package_licenses` splits by `' '` but some of these licenses are true license expressions using 'and' and 'or' (just not in caps).

OK, we can look at a more sophisticated method of weeding out true license expressions and old ones later. For now, can you move this change into the get_package_licenses function?